pystats_docs.md

```diff
@@ -0,0 +1,81 @@
+# pystats documentation
+
+This describes the meaning of the various sections and fields in the pystats output.
+
+## Execution counts
+
+This is the count of how many times each Tier 1 instruction is executed.
+
+The "miss ratio" column shows the percentage of times the instruction executed that it deoptimized.
+
+## Pair counts
+
+Shows the Top 100 most frequently occurring pairs of instructions as executed.
+
+## Predecessor / Successor pairs
+
+Shows the top 5 instructions that are executed before and after each instruction.
+
+## Specialization stats
+
+TBD
+
+## Specialization effectiveness
+
+TBD
+
+## Call stats
+
+TBD
+
+## Object stats
+
+TBD
```

Comment:
Grab bag of stats about objects:

"Allocations" means "allocations that are not from a freelist". Total allocations = "Allocations from freelist" + "Allocations"

"New values" is the number of values array created for objects with managed dicts.

The cache hit/miss numbers are for the MRO cache, split into dunder and other names.